data/remnant/remnant.txt

```diff
@@ -276,6 +276,490 @@ phrase "remnant"
+	word
+		"."
+		"!"
+
+phrase "friendly remnant"
+	word
+		"If"
+	word
+		" "
+	word
+		"you have"
+		"the prefects gave you"
+		"our elders allowed you"
+		"our government granted you"
+	word
+		" "
+	word
+		"a license"
+		"a permit"
+		"permission to buy our outfits"
+	word
+		","
+	word
+		" "
+	word
+		"I"
+		"our experts"
+		"our veterans"
+		"my friends"
+	word
+		" "
+	word
+		"would suggest"
+		"would recommend"
+	word
+		" "
+	word
+		"buying"
+		"getting"
+	word
+		" "
+	phrase
+		"remnant inhibitor cannon"
+		"remnant thrasher cannon"
+		"remnant emp torpedo bay"
+		"remnant salvage scanner"
+	word
+		"They're the best!"
+		"You should definitely try it!"
+		"Definitely better than those outfits in your space."
+
+phrase "remnant inhibitor cannon"
+	word
+		"some"
+		"those"
+		"our"
+	word
+		" "
+	word
+		"inhibitor cannons."
+	word
+		" "
+	word
+		"They"
+		"Those weapons"
+	word
+		"can"
+		"will"
+	word
+		" "
+	word
+		"disrupt"
+		"disable"
+		"slow"
+	word
+		" "
+	word
+		"the enemy's thrusters"
+		"your target's steering"
+		"any engine"
+	word
+		" "
+	word
+		"and prevent them from escaping"
+		", and anyone won't stand a chance"
+	word
+		"."
+		"!"
+
+phrase "remnant thrasher cannon"
+	word
+		"a few"
+		"several"
+		"some"
+	word
+		" "
+	word
+		"thrasher cannons"
+		"thrasher guns"
+		"thrasher turrets"
+	word
+		"."
+	word
+		" "
+	word
+		"Compared to"
+		"When placed beside"
+	word
+		" "
+	word
+		"your energy blasters,"
+	word
+		" "
+	word
+		"they're more efficient in terms of heat"
+		"you can deal more damage with our cannons"
+		"they may be consuming more energy, but they have better heat and damage"
+		"our cannons may be more expensive, but they have better damage"
+		"they're larger, but they can deal more damage and release less heat"
+		"they're a force to reckoned with"
+	word
+		"."
+		"!"
+
+phrase "remnant emp torpedo bay"
+	word
+		"an EMP Torpedo Bay."
+	word
+		" "
+	word
+		"With a single shot, it"
+		"Firing one"
+		"One torpedo"
+	word
+		" "
+	word
+		"will"
+		"can"
+	word
+		" "
+	word
+		"disrupt"
+		"disable"
+		"deplete"
+	word
+		" "
+	word
+		"the enemy's systems"
+		"your target's batteries"
+		"your enemy's shields"
+		"the target's engines"
+	word
+		"."
+		"!"
+
+phrase "remnant salvage scanner"
+	word
+		"one or two"
+		"a dozen"
+		"some"
+	word
+		" "
+	word
+		"salvage scanners."
+	word
+		" "
+	word
+		"They combine outfit and tactical scanners into one package"
+		"Compared to your separated versions, our special scanner is more powerful"
+		"They have a longer range than your outfit scanner"
+		"Those babies scan fast"
+		"They're a must-have for plundering"
+	word
+		"."
+		"!"
+
+phrase "friendly remnant"
+	word
+		"Have you heard"
+		"Do you know"
+		"You know anything"
+	word
+		" "
+	word
+		"about"
+	word
+		" "
+	word
+		"the"
+		"those"
+	word
+		" "
+	word
+		"Quarg"
+		"Pug"
+		"Hai"
+		"Drak"
+		"Archons"
+		"insects far west of our space"
```

Comment:
If this is suppose to be referring to the Coalition, I'd expect that the Remnant would know their name instead of just referring to them as insects. Simply saying "Coalition" doesn't line up with the previous word "those," though, and I don't think it makes much sense to name the members of the Coalition separately. Might be a good place to use the `replace` function.

Reply:
It basically comes down to scanners. They're probably more likely to know that there are beings that look like insects than to know that they formally identify themselves as the coalition.